python/cuml/test/test_umap.py

```diff
@@ -276,9 +276,12 @@ def test_umap_fit_transform_against_fit_and_transform():
     assert joblib.hash(ft_embedding) != joblib.hash(fit_embedding_diff_input)
 
 
-@pytest.mark.parametrize('n_components', [2, 13])
+@pytest.mark.parametrize('n_components', [2, 10, 21, 23, 24, 25, 26,
```

Comment:
We should perhaps sample from this set and use only 2 or 3 so we don't slow down the pytests too much.